### ISSUE TITLE
Error with track_lookup.

### ISSUE THREAD

**Issue by Icesenator** [Flexget/Flexget#1394]
#### Config:
```
  filmsSearch:
    discover:
      what:
       - movie_list: all
      from:
       - piratebay:
           category: video
           sort_by: seeds
       - t411:
           category: "Film/Vidéo"
    regexp:
      reject:
        - VOSTFR
        - Multisub
        - Multi-Subs
        - fri
        - free
        - fro
      accept:
        - FR
        - FRENCH
        - MULTi
    quality: 1080i+
    trakt_lookup:
      username: Xobble
    if:
      - trakt_collected : reject
    content_filter:
      require:
        - '*.avi'
        - '*.mkv'
      require_mainfile: yes
    content_size:
      max: 15000
    torrent_alive:
      min_seeds: 1
      reject_for: 30 minutes
```
  
#### Log:
```
2016-09-07 18:40 ERROR    lazy_lookup   filmsSearch     Unhandled error in lazy lookup plugin
2016-09-07 18:40 CRITICAL manager       filmsSearch     An unexpected crash has occurred. Writing crash report to /home/disk2/icesenator/crash_report.2016.09.07.184032156159.log. Please verify you are running the latest version of flexget by using "flexget -V" from CLI or by using version_checker plugin at http://flexget.com/wiki/Plugins/version_checker. You are currently using version 2.3.13
```
Detail : 
```
2016-09-07 18:39 ERROR    lazy_lookup   filmsSearch     Unhandled error in lazy lookup plugin
Traceback (most recent call last):
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/flexget/utils/lazy_dict.py", line 37, in __getitem__
    func(self.store)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/flexget/plugins/metainfo/trakt_lookup.py", line 248, in lazy_movie_lookup
    movie = lookup_movie(**lookupargs)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/flexget/utils/database.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/flexget/plugins/internal/api_trakt.py", line 1142, in lookup_movie
    if not session.query(TraktMovieSearchResult).filter(TraktMovieSearchResult.search == title.lower()).first():
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2659, in first
    ret = list(self[0:1])
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2457, in __getitem__
    return list(res)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2760, in __iter__
    self.session._autoflush()
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/session.py", line 1293, in _autoflush
    self.flush()
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/session.py", line 2019, in flush
    self._flush(objects)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/session.py", line 2137, in _flush
    transaction.rollback(_capture_exception=True)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/util/langhelpers.py", line 60, in __exit__
    compat.reraise(exc_type, exc_value, exc_tb)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/session.py", line 2101, in _flush
    flush_context.execute()
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/unitofwork.py", line 373, in execute
    rec.execute(self)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/unitofwork.py", line 487, in execute
    self.dependency_processor.process_saves(uow, states)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/dependency.py", line 1092, in process_saves
    secondary_update, secondary_delete)
  File "/home/disk2/icesenator/flexget/local/lib/python2.7/site-packages/sqlalchemy/orm/dependency.py", line 1113, in _run_crud
    result.rowcount)
StaleDataError: DELETE statement on table 'trakt_image_movie' expected to delete 1 row(s); Only 3 were matched.
```

**Comment by cvium** [comment]
This should've been fixed in version 2.3.18. Reopen if it still persists.

**Comment by Icesenator** [comment]
Yeah ! Good Job ! Thks you !

Le dim. 18 sept. 2016 à 12:42, Claus Vium <notifications@github.com> a
écrit :
